tests/testdata/version.txt

```diff
@@ -1 +1 @@
-a8366f29816e0ea6665131f241d916ee59904524
+ce5623976f130d44e6ff41fab057a321f0272929
```

Comment:
Why not use the version tag here?

Reply:
good point, https://github.com/babylonlabs-io/babylon-sdk/pull/46 fixed this